portal/sign/assets/signer_modal.php

```diff
@@ -10,8 +10,8 @@
  * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
 
-require_once(__DIR__ . "/../../../src/Common/Session/SessionUtil.php");
-OpenEMR\Common\Session\SessionUtil::portalSessionStart();
+//require_once(__DIR__ . "/../../../src/Common/Session/SessionUtil.php");
+//OpenEMR\Common\Session\SessionUtil::portalSessionStart();
```

Comment:
this makes me nervous and will mean that in line 16 below the $_SESSION['portal_init'] will never be captured (not sure for what use cases it is needed) to flag that the script is being used by the portal.

Reply:
yes revert this...